source/features/parse-backticks.tsx

```diff
@@ -41,6 +43,40 @@ function init(): void {
 			parseBackticks(element);
 		}
 	});
+
+	// `isRepoSearch`
+	observe('#issue_search_results .f4:not(.rgh-backticks-already-parsed)', {
+		add(element) {
+			const child = element.firstElementChild!;
+			const keywords = [...child.querySelectorAll('em')].map(element => element.textContent);
+			// Combine text content to enable backticks parsing
+			child.textContent = `${child.textContent!}`;
+
+			// Parse backticks
+			element.classList.add('rgh-backticks-already-parsed');
+			parseBackticks(element);
+
+			// Rehighlight search keywords
+			for (const keyword of keywords) {
```

Comment:
This part is obviously borrowed from the two backtick parsing functions, just want to say they are clean and effective ❤